index.html

```diff
@@ -21,7 +21,7 @@ <h2 class="animate__animated fanimate__adeInUp">
 
     </div>
 
-    <svg class="hero-waves" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 24 150 28 " preserveAspectRatio="none">
+    <svg class="hero-waves" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 24 150 28 " preserveAspectRatio="none" alt="Decorative waves">
```

Comment:
Mhh... alt doesn't seem to be what is to be expected here. webaccessibility.com [still complains](https://www.webaccessibility.com/results/?url=https%3A%2F%2Friot-os-riot-os-org-preview-38.surge.sh%2F).

Reply:
https://stackoverflow.com/a/4756461/11921757

Reply:
The `aria-label` seemed to fix this.